pkg/loader/loader.go

```diff
@@ -60,19 +63,24 @@ type ModulesLoader interface {
 
 	// GetLoaded returns a mapping of loaded module paths to their text context.
 	GetLoaded() map[string]string
+
+	// GetLoadedModuleVersion returns the module version given the module name.
+	GetLoadedModuleVersion(moduleName string) string
```

Comment:
Can you change this to `GetLoadedModule()` and return a `Module`? The version will be an attribute of that, but the function would have broader use.

Reply:
Done